src/ripple/app/tx/applySteps.h

```diff
@@ -82,32 +106,43 @@ struct PreclaimResult
     {
     }
 
+    /// Constructor
     template<class Context>
     PreclaimResult(Context const& ctx_,
         std::pair<TER, std::uint64_t> const& result)
         : PreclaimResult(ctx_, result.first, result.second)
     {
     }
 
+    /// Deleted copy assignment operator
     PreclaimResult& operator=(PreclaimResult const&) = delete;
 };
 
+/** Structure describing the account consequences if
```

Comment:
How should I understand  `account consequences` if the class is called `TxConsequences`?